scripts/install/setup_properties.sh

```diff
@@ -91,20 +92,21 @@ export HALYARD_VERSION=1.20.2
 
 # The specified network must exist, and it must not be a legacy network.
 # More info on legacy networks can be found here: https://cloud.google.com/vpc/docs/legacy
-export NETWORK=default
+export NETWORK=$NETWORK
 
 # If cluster does not exist, it will be created.
 export GKE_CLUSTER=${GKE_CLUSTER:-\$DEPLOYMENT_NAME}
 export ZONE=$ZONE
 export REGION=$REGION
-
+export SUBNET=$SUBNET
+export SPINNAKER_VERSION=1.14.5
+export HALYARD_VERSION=1.20.2
 # These are only considered if a new GKE cluster is being created.
 export GKE_CLUSTER_VERSION=1.12.7
 export GKE_MACHINE_TYPE=n1-highmem-4
 export GKE_DISK_TYPE=pd-standard
 export GKE_DISK_SIZE=100
 export GKE_NUM_NODES=3
-
```

Comment:
Accidentally removed this blank line? I think it helped with readability to break up the vars this way.

Reply:
I added back the blank line